platform/ios/platform/ios/scripts/swift_package_template.swift

```diff
@@ -6,14 +6,21 @@ let package = Package(
     products: [
         .library(
             name: "Mapbox",
-            targets: ["Mapbox"])
+            targets: ["Mapbox"]),
+        .library(
+            name: "MetalANGLE",
+            targets: ["MetalANGLE"])            
     ],
     dependencies: [
     ],    
     targets: [
         .binaryTarget(
             name: "Mapbox",
             url: "MAPBOX_PACKAGE_URL",
-            checksum: "MAPBOX_PACKAGE_CHECKSUM")
+            checksum: "MAPBOX_PACKAGE_CHECKSUM"),
+        .binaryTarget(
+            name: "MetalANGLE",
+            url: "METAL_ANGLE_PACKAGE_URL",
+            checksum: "METAL_ANGLE_PACKAGE_CHECKSUM")            
```

Comment:
For future reviewers (assume Petr already knows this)

Please see notes at https://github.com/maplibre/maplibre-gl-native-distribution/pull/5 on how to test locally built `*.xcframeworks` with SPM.